akka-actor-typed-tests/src/test/scala/docs/akka/typed/StyleGuideDocExamples.scala

```diff
@@ -371,18 +371,23 @@ object StyleGuideDocExamples {
     //#public-private-messages-2
     // above example is preferred, but this is possible and not wrong
     object Counter {
-      sealed trait PrivateCommand
-      sealed trait Command extends PrivateCommand
+      // The type of all public and private messages the Counter actor handles
+      sealed trait Message
+
+      /** Counter's public message protocol type. */
+      sealed trait Command extends Message
       case object Increment extends Command
       final case class GetValue(replyTo: ActorRef[Value]) extends Command
       final case class Value(n: Int)
 
-      // Tick is a PrivateCommand so can't be sent to an ActorRef[Command]
-      case object Tick extends PrivateCommand
+      // The type of the Counter actor's internal event messages.
+      private sealed trait Event extends Message
```

Comment:
It's unfortunate to use the name `Event` for those since that name is heavily used in event sourcing and there it's something different, and not limited to internal (private) messages.

`PrivateCommand` would be better.